src/ui/buttons/buttons.jsx

```diff
@@ -179,6 +179,8 @@ export function Buttons(props: ButtonsProps): ElementNode {
     supportedNativeBrowser,
     showPayLabel,
     displayOnly,
+    // eslint-disable-next-line no-unused-vars
+    message,
```

Comment:
curious, why do we need this here if we aren't using it?

Reply:
we aren't using it _yet_, but it will be the jumping-off point for use in another story in this feature, to be added soon.